autode/species/molecule.py

```diff
@@ -22,7 +22,7 @@ def _init_smiles(self, smiles):
         """Initialise a molecule from a SMILES string using RDKit if it's
         purely organic"""
 
-        if any(metal in smiles for metal in metals):
+        if any(f'[{metal}]' in smiles for metal in metals):
```

Comment:
What was wrong with this before? This means metals with charges etc won't be detected

Reply:
see first comment. & that is a very good point – is this a regex situation..?

Reply:
`\[.*?\]` perhaps

Reply:
my regex is...non existent...if you say so ;)